domene/src/main/java/no/nav/foreldrepenger/mottak/task/joark/SendInnEndeligJournalførtTask.java

```diff
@@ -0,0 +1,158 @@
+package no.nav.foreldrepenger.mottak.task.joark;
+
+import static java.lang.String.format;
+import static no.nav.foreldrepenger.fordel.kodeverdi.Journalstatus.JOURNALFOERT;
+import static no.nav.foreldrepenger.fordel.kodeverdi.Tema.FORELDRE_OG_SVANGERSKAPSPENGER;
+import static no.nav.foreldrepenger.mottak.felles.MottakMeldingDataWrapper.ARKIV_ID_KEY;
+import static no.nav.foreldrepenger.mottak.task.xml.MeldingXmlParser.erXmlMedKjentNamespace;
+import static no.nav.foreldrepenger.mottak.task.xml.MeldingXmlParser.unmarshallXml;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import no.nav.foreldrepenger.fordel.kodeverdi.DokumentTypeId;
+import no.nav.foreldrepenger.fordel.kodeverdi.Journalposttype;
+import no.nav.foreldrepenger.mottak.felles.MottakMeldingDataWrapper;
+import no.nav.foreldrepenger.mottak.felles.WrappedProsessTaskHandler;
+import no.nav.foreldrepenger.mottak.journal.ArkivTjeneste;
+import no.nav.foreldrepenger.mottak.person.PersonInformasjon;
+import no.nav.foreldrepenger.mottak.task.VLKlargjørerTask;
+import no.nav.foreldrepenger.mottak.tjeneste.ArkivUtil;
+import no.nav.vedtak.exception.TekniskException;
+import no.nav.vedtak.exception.VLException;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTask;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskTjeneste;
+import no.nav.vedtak.felles.prosesstask.api.TaskType;
+
+/**
+ * <p>
+ * ProssessTask som håndterer uthenting av saksinformasjon fra Journalarkivet(joark).
+ * Henter journalpost / metadata og original (XML) dokument og finner destinasjon for journalposten
+ * </p>
+ */
+@ApplicationScoped
+@ProsessTask(SendInnEndeligJournalførtTask.TASKNAME)
+public class SendInnEndeligJournalførtTask extends WrappedProsessTaskHandler {
+
+    static final String TASKNAME = "fordeling.sendInnJournalpost";
+
+    private static final TaskType TASK_KLARGJØR = TaskType.forProsessTask(VLKlargjørerTask.class);
+
+    private static final Logger LOG = LoggerFactory.getLogger(SendInnEndeligJournalførtTask.class);
+
+    private PersonInformasjon pdl;
+    private ArkivTjeneste arkiv;
+
+    public SendInnEndeligJournalførtTask() {
+
+    }
+
+    @Inject
+    public SendInnEndeligJournalførtTask(ProsessTaskTjeneste taskTjeneste, PersonInformasjon pdl, ArkivTjeneste arkiv) {
```

Comment:
Når har man behov for denne?

Reply:
Se på siste versjon av PR. Den finnes ikke lenger

Reply:
Jeg tenkte egentlig på denne sendInnPåSak(ArkivJournalpost journalpost, String saksnummer) - det er kanskje tilfeller hvor journalposten er ferdigstilt av Gosys, men ikke koblet til sak hos oss?